### PR TITLE
Added logs for postgresql container

### PR DESCRIPTION
**Pull request by ethervoid**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -4,7 +4,6 @@
   "postgres": {
     "image": "carto/bigmetadata_postgres",
     "volumes": [
-      "./postgres/postgresql.conf:/etc/postgresql/9.5/main/postgresql.conf",
```
Comment (javitonino) [inline docker-compose.yml]: If you remove it, how does it know to mount the postgres configuration file?

Comment (ethervoid) [reply docker-compose.yml]: It [copies the file](https://github.com/CartoDB/bigmetadata/pull/312/files#diff-919c1cc4045b5a538c5600a717a3baabR26) when you create the image of the container. From my PoV having a shared configuration file that could change is not a good idea, containers should be efemeral if we need to change the config, we create a new image and that's it

Comment (javitonino) [reply docker-compose.yml]: Oh, so COPY loads from outside the docker. I did not know that, I thought it was looking inside the image. 👍 then

Comment (ethervoid) [reply docker-compose.yml]: Yep it does the copy operation at building time

**Pull request merged**